cms/plugin_pool.py

```diff
@@ -37,10 +37,67 @@ def clear(self):
         self.plugins = {}
         self.patched = False
 
+    def validate_templates(self, plugin=None):
+        """
+        Plugins templates are validated at this stage
+
```

Comment:
This is actually quite old code. I prefer to import this as-is, and cleanup it in 3.3